File: Data Structures & Algorithms/car-fleet/submission-1.java
class Solution {
    public int carFleet(int target, int[] position, int[] speed) {
        Stack<Double> ans=new Stack<>();
        int size=position.length;
        int[][] cars=new int[size][2];
        for(int i=0;i<size;i++){
            cars[i][0]=position[i];
            cars[i][1]=speed[i];
        }
        Arrays.sort(cars,(a,b)->Integer.compare(b[0],a[0]));

        int n=cars.length;

        for(int i=0;i<n;i++){
            double time=(double)(target-cars[i][0])/cars[i][1];
             if(ans.isEmpty()||time>ans.peek())
                ans.push(time);
          }
        return ans.size();
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ans=[]
        size=len(nums)
        for i in range(0,size-1):
            for j in range(i+1,size):
                if(nums[i]+nums[j]==target):
                    ans.append(i)
                    ans.append(j)
                    break
        return ans
